mission_control/src/behaviors/abort_behavior.cpp

```diff
@@ -33,51 +33,57 @@
  *********************************************************************/
 
 // Original version: Christopher Scianna Christopher.Scianna@us.QinetiQ.com
-#include <string>
-
 #include "mission_control/behaviors/abort_behavior.h"
 
+#include <string>
+
 using mission_control::AbortBehavior;
 
-AbortBehavior::AbortBehavior(const std::string& name,
-                                             const BT::NodeConfiguration& config)
-    : Behavior(name, config)
+AbortBehavior::AbortBehavior(const std::string& name, const BT::NodeConfiguration& config)
+    : BT::ActionNodeBase(name, config)
 {
-  subStateData_ =
-      nodeHandle_.subscribe("/state", 1, &AbortBehavior::stateDataCallback, this);
+  subStateData_ = nodeHandle_.subscribe("/state", 1, &AbortBehavior::stateDataCallback, this);
```

Comment:
@patapufate nit^2: it'd be nice to subscribe when necessary only :)

Reply:
It was implemented same as Waypoint behavior.
3f4c949